### PR TITLE
fix(adminapi): system config save always failed (#583)

### LINKED ISSUE #583: [Bug] 系统配置->保存配置 永远失败
哪怕初始安装，第一次进入，去系统配置里点击保存配置都是失败。如图。

Image

### PR DESCRIPTION
**Pull request by jamiesun**
## Summary
- Fix `parsePagination` to accept React Admin's `perPage` (in addition to `pageSize`) and **clamp** oversized sizes instead of silently falling back to 20.
- Allow empty string values on settings create/update so optional configs (EAP certs, LDAP password, etc.) can be saved/cleared.
- Add regression tests covering pagination aliases and empty-value create/update.

### Root cause (#583)
The system config page loads settings with `perPage=1000`, but the backend only read `pageSize` and defaulted to **20**. With 24 schema-backed settings (LDAP group pushed the count past 20), the last items had no `id` in the UI; save tried POST create → 409 conflict / 400 empty value → "配置保存失败" every time.

## Test plan
- [x] `go test ./internal/adminapi/ -count=1`
- [x] `golangci-lint run ./internal/adminapi/`
- [ ] Fresh install → System Config → Save without edits → success
- [ ] Edit an optional empty field (e.g. LDAP password / EAP cert) → Save → success
- [ ] Confirm list returns all settings when called with `?page=1&perPage=1000`

Fixes #583

### CODE REVIEW

File: internal/adminapi/settings.go [hunk 1]
```diff
@@ -213,21 +226,21 @@ func updateSettings(c echo.Context) error {
...
 	if payload.Type != "" {
 		setting.Type = strings.TrimSpace(payload.Type)
 	}
 	if payload.Name != "" {
 		setting.Name = strings.TrimSpace(payload.Name)
 	}
-	if payload.Value != "" {
-		setting.Value = payload.Value
+	if payload.Value != nil {
+		setting.Value = *payload.Value
```
[the remark this answers is not in the document]
Comment (jamiesun) [reply internal/adminapi/settings.go:244]: Addressed in the follow-up commit: create/update now call `ConfigManager.ValidateValue` against the registered schema. Empty values stay valid for optional string keys (e.g. `radius.EapTlsServerCert`), and empty/invalid int/bool/duration/json values are rejected with `INVALID_VALUE`. Added unit + API regression coverage.
